setup.py

```diff
@@ -248,9 +248,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     "requests>=2.26.0",
 ]
 doc = [
-    # Astroid 2.12.* breaks documentation building
-    # We can remove the limit here after https://github.com/PyCQA/astroid/issues/1708 is solved
-    "astroid<2.12.0",
+    "astroid",
```

Comment:
```suggestion
    "astroid>=2.12.3",
```
Don't we need to bump to the version that contains the fix?
or at least skip the ones known to have the issue: `!=2.12.0, !=2.12.1, !=2.12.2` ?

Reply:
done